### PR TITLE
Organization Members CRUD/Profile View

### PR DESCRIPTION
**Pull request by lukasbischof**
# Organization Members CRUD

### Result

image
image

### Todo

- [ ] Prevent deletion of own account
- [ ] Specs

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -63,10 +63,15 @@ fr:
         workfree_days:
           one: Jour de congé
           other: jours de congé
-      organizatino_holiday:
```
Comment (lukasbischof) [inline config/locales/fr.yml:66]: Typo

File: app/views/organizations/organization_members/index.html.slim [hunk 2]
```diff
@@ -0,0 +1,2 @@
+h1 Organizations::OrganizationMembers#index
```
Comment (TheFehr) [inline app/views/organizations/organization_members/index.html.slim:1]: Destroy, index and update files seem pretty default?

Comment (lukasbischof) [reply app/views/organizations/organization_members/index.html.slim:1]: whops pr should have been a draft

File: app/controllers/organizations/organization_members_controller.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      :first_name, :last_name, :phone, :organization_role,
+      user_attributes: %i[email id]
+    ].freeze
+
+    before_action :set_organization_member, only: %i[edit update destroy]
+
+    helper UsersHelper
+
+    def index
+      @organization_members = helpers.current_organization_admin.organization.organization_members.includes(:user)
```
Comment (TheFehr) [inline app/controllers/organizations/organization_members_controller.rb:15]: What is the usage of the `.includes(:user)`?

Comment (lukasbischof) [reply app/controllers/organizations/organization_members_controller.rb:15]: Preloading

File: app/views/organizations/organization_members/edit.html.slim [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            = f.input :first_name, required: true
+        .col-lg-6
+          .form-group data-children-count="1"
+            = f.input :last_name
+        .col-lg-6
+          .form-group data-children-count="1"
+            - if f.object.user.present?
+              = f.simple_fields_for :user do |ff|
+                = ff.input :email
+                - if ff.object.unconfirmed_email.present?
```
Comment (TheFehr) [inline app/views/organizations/organization_members/edit.html.slim:22]: Maybe not use f and ff. Maybe something like user_f(orm) for the inner one?

File: spec/requests/organizations/organization_members_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+        perform_request
+      end
+
+      it 'returns a list of organization members inside the organization' do
+        expect(response).to be_successful
+        expect(response.body).to include(*organization.organization_members.pluck(:first_name))
+        expect(response.body).not_to include(
+          outside_organization_member.first_name,
+          outside_organization_member.last_name
+        )
```
Comment (TheFehr) [inline spec/requests/organizations/organization_members_controller_spec.rb:36]: Why u spec only first_name from inside organization member but first and last from outside?

**Pull request merged**